trajectory_verifier/CheckTrajectoryValidityQuery query
---
trajectory_verifier/CheckTrajectoryValidityResult result
